### PR TITLE
Downcase 'Area' in 'Policy Area' dropdown label

### PR DESCRIPTION
**Pull request by jackscotti**
Content edit requested by John Turnbull (Content team).

### CODE REVIEW

**Comment by rboulton** [comment]
:+1:

File: features/step_definitions/filtering_steps.rb [hunk 1]
```diff
@@ -68,7 +68,7 @@
   assert_listed_document_count 1
   assert page.has_content? "Guidance publication"
 
-  select_filter "Policy Area", "A Topic", and_clear_others: true
+  select_filter "Policy area", "A Topic", and_clear_others: true
```
Comment (mattbostock) [inline features/step_definitions/filtering_steps.rb]: Should 'A Topic' be 'A topic'?

File: features/step_definitions/filtering_steps.rb [hunk 2]
```diff
@@ -177,7 +177,7 @@
   within '#document-filter' do
     page.fill_in "Contains", with: "keyword"
     page.select "News stories", from: "Announcement type"
-    page.select "A Topic", from: "Policy Area"
+    page.select "A Topic", from: "Policy area"
```
Comment (mattbostock) [inline features/step_definitions/filtering_steps.rb]: Ditto above 'A Topic'

**Comment by jackscotti** [comment]
@mattbostock, It's selecting an element called 'A Topic' from a 'Policy area' drop-down. Due to the huge number of files containing the word Topic we decided to proceed first by only renaming the labels in the UI.

There'll be a follow-up ticket that will request us to renaming topic to policy area also in the back-end. That'll probably be the best time for getting rid of all the instances of the word Topic.

**Comment by mattbostock** [comment]
:+1: Thanks for explaining @jackscotti.

**Pull request merged**